src/python/turicreate/toolkits/_model.py

```diff
@@ -115,7 +115,17 @@ def load_model(location):
                     model = _extensions.style_transfer()
                     model.import_from_custom_model(model_data, model_version)
                     return cls(model)
-                    
+
+                if name=='one_shot_object_detector' and OD_USE_CPP:
+                    od_cls = MODEL_NAME_MAP['object_detector']
```

Comment:
What class is `od_cls`? Why do we need something different from `cls`?

Reply:
Oh `cls` here is an one-shot-object_detector model. It contains a object_detector model inside the class, which I refer to `od_cls` here.

Reply:
`od_cls` is always an `Object_detector_beta` object.